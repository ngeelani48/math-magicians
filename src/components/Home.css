.home-container {
  padding: 0 1%;
}

.home-title {
  padding: 1%;
}

.homeParas {
  padding: 1%;
  font-size: 1.1rem;
}
