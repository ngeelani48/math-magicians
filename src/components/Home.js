import './Home.css';

const Home = () => (
  <div className="home-container">
    <h2 className="home-title">Welcome to my Math Magicians website!</h2>
    <div className="homeParas">
      <p className="homeP">
        Lorem ipsum dolor sit, amet consectetur adipisicing elit.
        Facilis minima explicabo ex laboriosam vitae quas repudiandae culpa quam labore omnis est,
        laudantium enim in nulla eius inventore neque odit eum!
        Lorem ipsum dolor sit, amet consectetur adipisicing elit.
        Facilis minima explicabo ex laboriosam vitae quas repudiandae culpa quam labore omnis est,
        laudantium enim in nulla eius inventore neque odit eum!
        Lorem ipsum dolor sit, amet consectetur adipisicing elit.
        Facilis minima explicabo ex laboriosam vitae quas repudiandae culpa quam labore omnis est,
        laudantium enim in nulla eius inventore neque odit eum!
        Lorem ipsum dolor sit, amet consectetur adipisicing elit.
        Facilis minima explicabo ex laboriosam vitae quas repudiandae culpa quam labore omnis est,
        laudantium enim in nulla eius inventore neque odit eum!
      </p>
      <p className="homeP">
        Lorem ipsum dolor sit, amet consectetur adipisicing elit.
        Facilis minima explicabo ex laboriosam vitae quas repudiandae culpa quam labore omnis est,
        laudantium enim in nulla eius inventore neque odit eum!
        Lorem ipsum dolor sit, amet consectetur adipisicing elit.
        Facilis minima explicabo ex laboriosam vitae quas repudiandae culpa quam labore omnis est,
        laudantium enim in nulla eius inventore neque odit eum!
        Lorem ipsum dolor sit, amet consectetur adipisicing elit.
        Facilis minima explicabo ex laboriosam vitae quas repudiandae culpa quam labore omnis est,
        laudantium enim in nulla eius inventore neque odit eum!
        Lorem ipsum dolor sit, amet consectetur adipisicing elit.
        Facilis minima explicabo ex laboriosam vitae quas repudiandae culpa quam labore omnis est,
        laudantium enim in nulla eius inventore neque odit eum!
      </p>
    </div>
  </div>
);

export default Home;
