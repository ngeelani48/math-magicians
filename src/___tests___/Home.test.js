import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../components/Home';

it('check if the Home section renders properly', () => {
  const tree = renderer.create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
});
