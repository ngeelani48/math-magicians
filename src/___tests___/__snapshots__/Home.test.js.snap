// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`check if the Home sectio renders properly 1`] = `
<div
  className="home-container"
>
  <h2
    className="home-title"
  >
    Welcome to my Math Magicians website!
  </h2>
  <div
    className="homeParas"
  >
    <p
      className="homeP"
    >
      Lorem ipsum dolor sit, amet consectetur adipisicing elit. Facilis minima explicabo ex laboriosam vitae quas repudiandae culpa quam labore omnis est, laudantium enim in nulla eius inventore neque odit eum! Lorem ipsum dolor sit, amet consectetur adipisicing elit. Facilis minima explicabo ex laboriosam vitae quas repudiandae culpa quam labore omnis est, laudantium enim in nulla eius inventore neque odit eum! Lorem ipsum dolor sit, amet consectetur adipisicing elit. Facilis minima explicabo ex laboriosam vitae quas repudiandae culpa quam labore omnis est, laudantium enim in nulla eius inventore neque odit eum! Lorem ipsum dolor sit, amet consectetur adipisicing elit. Facilis minima explicabo ex laboriosam vitae quas repudiandae culpa quam labore omnis est, laudantium enim in nulla eius inventore neque odit eum!
    </p>
    <p
      className="homeP"
    >
      Lorem ipsum dolor sit, amet consectetur adipisicing elit. Facilis minima explicabo ex laboriosam vitae quas repudiandae culpa quam labore omnis est, laudantium enim in nulla eius inventore neque odit eum! Lorem ipsum dolor sit, amet consectetur adipisicing elit. Facilis minima explicabo ex laboriosam vitae quas repudiandae culpa quam labore omnis est, laudantium enim in nulla eius inventore neque odit eum! Lorem ipsum dolor sit, amet consectetur adipisicing elit. Facilis minima explicabo ex laboriosam vitae quas repudiandae culpa quam labore omnis est, laudantium enim in nulla eius inventore neque odit eum! Lorem ipsum dolor sit, amet consectetur adipisicing elit. Facilis minima explicabo ex laboriosam vitae quas repudiandae culpa quam labore omnis est, laudantium enim in nulla eius inventore neque odit eum!
    </p>
  </div>
</div>
`;

exports[`check if the Home section renders properly 1`] = `
<div
  className="home-container"
>
  <h2
    className="home-title"
  >
    Welcome to my Math Magicians website!
  </h2>
  <div
    className="homeParas"
  >
    <p
      className="homeP"
    >
      Lorem ipsum dolor sit, amet consectetur adipisicing elit. Facilis minima explicabo ex laboriosam vitae quas repudiandae culpa quam labore omnis est, laudantium enim in nulla eius inventore neque odit eum! Lorem ipsum dolor sit, amet consectetur adipisicing elit. Facilis minima explicabo ex laboriosam vitae quas repudiandae culpa quam labore omnis est, laudantium enim in nulla eius inventore neque odit eum! Lorem ipsum dolor sit, amet consectetur adipisicing elit. Facilis minima explicabo ex laboriosam vitae quas repudiandae culpa quam labore omnis est, laudantium enim in nulla eius inventore neque odit eum! Lorem ipsum dolor sit, amet consectetur adipisicing elit. Facilis minima explicabo ex laboriosam vitae quas repudiandae culpa quam labore omnis est, laudantium enim in nulla eius inventore neque odit eum!
    </p>
    <p
      className="homeP"
    >
      Lorem ipsum dolor sit, amet consectetur adipisicing elit. Facilis minima explicabo ex laboriosam vitae quas repudiandae culpa quam labore omnis est, laudantium enim in nulla eius inventore neque odit eum! Lorem ipsum dolor sit, amet consectetur adipisicing elit. Facilis minima explicabo ex laboriosam vitae quas repudiandae culpa quam labore omnis est, laudantium enim in nulla eius inventore neque odit eum! Lorem ipsum dolor sit, amet consectetur adipisicing elit. Facilis minima explicabo ex laboriosam vitae quas repudiandae culpa quam labore omnis est, laudantium enim in nulla eius inventore neque odit eum! Lorem ipsum dolor sit, amet consectetur adipisicing elit. Facilis minima explicabo ex laboriosam vitae quas repudiandae culpa quam labore omnis est, laudantium enim in nulla eius inventore neque odit eum!
    </p>
  </div>
</div>
`;

exports[`check if the calculator renders properly 1`] = `
<div
  className="layout"
>
  <h2
    className="calc-title"
  >
    Let's do some Math!
  </h2>
  <div
    className="container"
  >
    <div
      className="result-display"
    >
      0
    </div>
    <div
      className="calc-buttons"
    >
      <button
        className="calc-button"
        onClick={[Function]}
        type="button"
      >
        AC
      </button>
      <button
        className="calc-button"
        onClick={[Function]}
        type="button"
      >
        +/-
      </button>
      <button
        className="calc-button"
        onClick={[Function]}
        type="button"
      >
        %
      </button>
      <button
        className="calc-button-right"
        onClick={[Function]}
        type="button"
      >
        รท
      </button>
      <button
        className="calc-button"
        onClick={[Function]}
        type="button"
      >
        7
      </button>
      <button
        className="calc-button"
        onClick={[Function]}
        type="button"
      >
        8
      </button>
      <button
        className="calc-button"
        onClick={[Function]}
        type="button"
      >
        9
      </button>
      <button
        className="calc-button-right"
        onClick={[Function]}
        type="button"
      >
        x
      </button>
      <button
        className="calc-button"
        onClick={[Function]}
        type="button"
      >
        4
      </button>
      <button
        className="calc-button"
        onClick={[Function]}
        type="button"
      >
        5
      </button>
      <button
        className="calc-button"
        onClick={[Function]}
        type="button"
      >
        6
      </button>
      <button
        className="calc-button-right"
        onClick={[Function]}
        type="button"
      >
        -
      </button>
      <button
        className="calc-button"
        onClick={[Function]}
        type="button"
      >
        1
      </button>
      <button
        className="calc-button"
        onClick={[Function]}
        type="button"
      >
        2
      </button>
      <button
        className="calc-button"
        onClick={[Function]}
        type="button"
      >
        3
      </button>
      <button
        className="calc-button-right"
        onClick={[Function]}
        type="button"
      >
        +
      </button>
      <button
        className="calc-button-zero"
        onClick={[Function]}
        type="button"
      >
        0
      </button>
      <button
        className="calc-button"
        onClick={[Function]}
        type="button"
      >
        .
      </button>
      <button
        className="calc-button-right"
        onClick={[Function]}
        type="button"
      >
        =
      </button>
    </div>
  </div>
</div>
`;
